### PR TITLE
Linux Qt binding granurality

### PR DESCRIPTION
**Pull request by jakubjezek001**
MODIFIED BY @iLLiCiTiT 

## Changelog Description
Added `distro` python module to detect linux distribution name used for detection of Qt binding. Also added option to use PySide2 during docker build.

## Source issue
Rocky 8 might not support PySide6 by default and that might require to use PySide2 instead of PySide6. That was case because of specific rocky 8 state and GPU drivers. For now that is considered as edge case as most of machines should use PySide6 by rather.

### Error
```
qt.qpa.plugin: From 6.5.0, xcb-cursor0 or libxcb-cursor0 is needed to load the Qt xcb platform plugin.
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
```

### CODE REVIEW

**Comment by iLLiCiTiT** [comment]
We do the same for centos7 by adding `ENV QT_BINDING=centos7` in docker, so now we have 2 different approaches to that.

File: pyproject.toml [hunk 1]
```diff
@@ -46,6 +46,7 @@ semver = "^2.13.0"
 QtPy = "^2.3.0"
 pywin32 = { version = "301", markers = "sys_platform == 'win32'" }
 python3-xlib = { version="*", markers = "sys_platform == 'linux'"}
+distro = "^1.9.0"
```
Comment (iLLiCiTiT) [inline pyproject.toml:49]:
```suggestion
distro = { version="^1.9.0", markers = "sys_platform == 'linux'"}
```

File: poetry.lock [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
```
Comment (iLLiCiTiT) [inline poetry.lock:1]: We're forcing 1.8.1 on windows
```suggestion
# This file is automatically @generated by Poetry 1.8.1 and should not be changed by hand.
```

**Comment by iLLiCiTiT** [comment]
Modified the original PR to be able to build any linux installer with PySide2.

**Review by 64qam — APPROVED** [review]
works at client's end

**Pull request merged**